src/ticketmap/colors.js

```diff
@@ -1,18 +1,18 @@
 export const fillSection = (id, color, type = 'fill') => {
   if (!color) {
-    throw Error('Color is undefined for this section.');
+    throw Error('Color is undefined for this section.')
   }
-  getAllPaths(id).forEach(element => element.setAttribute(type, color));
+  getAllPaths(id).forEach(element => element.setAttribute(type, color))
 }
 
 export const setUnavailableColors = () => {
-  getAllPaths().forEach(element => element.setAttribute('data-unavailable-color', element.getAttribute('fill')));
+  getAllPaths().forEach(element => element.setAttribute('data-unavailable-color', element.getAttribute('fill')))
 }
 
 export const fillUnavailableColors = () => {
-  getAllPaths().forEach(element => element.setAttribute('fill', element.getAttribute('data-unavailable-color')));
+  getAllPaths().forEach(element => element.setAttribute('fill', element.getAttribute('data-unavailable-color')))
 }
 
 export const getAllPaths = (id) =>
-  Array.from(document.querySelectorAll(`[data-section-id${!!id ? `="${id}"` : ''}]`))
```

Comment:
Interesting, was this the formatter or was it a manual change?

Reply:
This was the work of `eslint --fix`